charter/opencdms_charter.md

```diff
@@ -29,11 +29,11 @@ For OpenCDMS to thrive, it needs to be seen to be agile and managed by consensus
 
 The OpenCDMS project will include modules providing specific data products to the WMO Information System, but assumes that WIS provides the interface to receive and transmit these products.
 
-The OpenCDMS project will include porting of end-user hydrology and climate applications which are currently operational in precursor systems (Climsoft, MCH, ClidDE, etc) but will not include development of new end-user applications. It is assumed that these will be provided by the (separate) WMO Climate Services Toolbox project, or others.
+The OpenCDMS project aims to offer hight quality hydrology and climate data, applications and services which help in adressing today national, regional and global needs. The OpenCDMS will ensure the porting of most functions in used by NMHSs, including the present open source CDMSs (Climsoft, MCH, ClidDE, etc.) and non-open source systems willing to participate to the project. It will also propose innovative methodologies and products that take into account available up-to-date standards and data. [This is because we need to attract also developed countries. They need to find an interest in the OpenCDMS project. Not to let them think that it is another system intended only for developing countries. It should be a system for all, open to all colaborative work on climate and hydrology]
 
 ## Project Approach
 
-OpenCDMS has two strands. One is a top-down approach which sets what is acceptable for the reference functions of OpenCDMS, where it will either replace existing CDMS or act as the reference for the processes in those CDMSs. The other is a bottom-up approach which builds on existing precursor CDMSs, including building rapid prototypes and adding recommended features of OpenCDMS into existing systems.
+OpenCDMS has two strands. One is a top-down approach which sets Towards which standards (practices, functions, methods, architecture, information technology) should OpenCDMS aim to achieve. The other is a bottom-up approach which builds on existing precursor CDMSs, including building rapid prototypes and adding recommended features of OpenCDMS into existing systems.
```

Comment:
I can see what this is saying, but I would rather not be specific. The original text was trying to explain the aims for both a practical operational CDMS and a reference system (as in the original WMO Resolution).